### ISSUE TITLE
Feature Request : - Sampler (DPM Solver) & ControlNet

### ISSUE THREAD

**Issue by shanmugad** [arenasys/qDiffusion#13]
**1. **Sampler Request :**** 

1. DPM Solver++
2. DPM Karras

**2. Concurrency Image Generation**

During Image generation, It take less than 15% of GPU (Colab) and it generate image one by one. Can you add an option it can generate multiple image simultaneously.

You can add an option where we user can select how many concurrency we want.

**3. **LoHA :**** 

[LoHA](https://github.com/arenasys/qDiffusion/issues/11#issuecomment-1603371203) is not supported under LoRA in qDiffusion. 

**4. ControlNet Supporting** 

[Many ControlNet ](https://github.com/arenasys/qDiffusion/issues/15#issuecomment-1608723075) is currently not supported by qDiffusion. Can you make it support wide rang of ControlNet

**Comment by arenasys** [comment]
1. I wont be supporting all the redundant samplers. Already DPM++ 2m, 2s a, SDE, 2m SDE with Regular, Karras and Exponential schedulers are supported.
2. `Batch size` is the parameter for increasing the number images generated in parallel. Concurrent generation would be impractical.
3. I wont be supporting all the redundant LyCORIS variants. LoHA is popular enough to warrant implementing, but its not a priority.
4. I hope to support all the ControlNet v1.1 models, support for more soon.

**Comment by shanmugad** [comment]
Thanks for replying @arenasys 

**Sampler :** Can you please add only `DPM Solver++` sampler, I use this sampler in website like Getimg and OpenArt. The output is really good and realistic. 

**Concurrency :** Just tested Batch size, it is not simultaneously (Tested with Google Colab). It create first image and than another image.

**Comment by arenasys** [comment]
I verified `Batch size` is working normally, try enable `Preview` (set it to `Light` etc) to see it more clearly. And note that `Batch count` will generate sequentially rather than in parallel. As for the `DPM Solver++` sampler, ill look into it, but know that the specific sampler isnt a huge factor for quality except for at low step count.

**Comment by shanmugad** [comment]
I can confirm `Batch size` working fine. And thanks for considering my request for `DPM Solver++` and `ControlNet`
